plugins/repository-s3/src/main/java/org/elasticsearch/cloud/aws/AwsS3Service.java

```diff
@@ -153,5 +153,6 @@
         Setting<String> ENDPOINT_SETTING = Setting.simpleString("cloud.aws.s3.endpoint", Property.NodeScope);
     }
 
-    AmazonS3 client(String endpoint, Protocol protocol, String region, String account, String key, Integer maxRetries);
+    AmazonS3 client(String endpoint, Protocol protocol, String region, String account, String key, Integer maxRetries,
+                    Boolean pathStyleAccess);
```

Comment:
Can we use a primitive instead of `Boolean`? And set it to `false` by default? So that we control the default value of the S3 plugin setting and not rely on the default value of the AWS SDK (which could be changed without we notice it).